src/ripple/protocol/jss.h

```diff
@@ -551,6 +564,7 @@ JSS(time);
 JSS(timeouts);                // out: InboundLedger
 JSS(track);                   // out: PeerImp
 JSS(traffic);                 // out: Overlay
+JSS(tokenid);                 // in: nft_sell_offers, nft_buy_offers
```

Comment:
rename to `nftid`

Reply:
Renamed to `nft_id` to match similar fields.  This is an API change, so we'll need to clear it with the authorities.  I made the change in a single commit so it will be easy to back out if necessary: https://github.com/ripple/rippled/pull/4101/commits/1a932bf835574686ca7f058640d498c509627899